README.md

```diff
@@ -8,6 +8,22 @@ The `AzureContainerAppsRC` task found in this repository is the _Release Candida
 of this Release Candidate task is to offer users immediate early access to features, bug fixes and patches that will
 eventually be rolled out in the official `AzureContainerApps` task at the end of every three week sprint.
 
+## Running this task on Microsoft-hosted agents
+
+If you are running this task on a
+[Microsoft-hosted agent](https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/hosted), you may find that this
+task is _not_ able to run successfully with the following operating systems:
+
+- macOS
+  - The [macOS runners](https://github.com/actions/runner-images#available-images) provided by Microsoft do not come
+  installed with Docker (more information [here](https://github.com/actions/runner-images/issues/17#issuecomment-614726536));
+  as a result, this task is not able to run any `docker` commands, such as pushing the built runnable application images
+  to ACR.
+- Windows
+  - The [Windows runners](https://github.com/actions/runner-images#available-images) provided by Microsoft comes with
```

Comment:
Not really a comment but more a question: so this is the only reason why we need Docker in this scenario: to pull images? I imagine that after that our buildpack-based approach obviously doesn't need Docker.

Reply:
That's correct -- the Microsoft-hosted Windows agents aren't able to _pull_ Linux-based images down, so anything to do with the Oryx builder would not work since it definitely needs our CLI image that is Debian-based. However, the scenario where users bring their own image to use will work (since that's just directly passed to the `az containerapp up` command) and the scenario where users bring their own Dockerfile _should_ work if the user doesn't include any Linux-based images in any of their layers.